docs/experiments/taint-propagators.md

```diff
@@ -6,7 +6,7 @@ description: "Taint propagators allow you to customize how taint is propagated."
 
 # Taint propagators
 
-Taint propagators allow you to customize how taint is propagated. 
+Taint propagators allow you to customize how taint is propagated. TODO
```

Comment:
@IagoAbal I made this PR so we can bring some of the ideas from the previous discussion here. Please, can we say a bit more here to specify taint propagators? We somehow specify them in action below, but I am still not sure if we state the essence. Maybe @efortuna will have an idea here also?

Reply:
yeah, this faces the problem of a circular definition: "don't define something with the same words as the actual thing" 

If you need inspiration, you can always look at page 5 in my script: https://docs.google.com/document/d/1hqB-a_wkqrEwk_ZsFnlZKOFo5HPeUdsGPMRj8umTDxk/edit

```suggestion
Taint propagators allow you to specify additional structures through which taint spreads, that Semgrep by default might not mark as tainted.
```